react-web/src/components/organisms/form/helpers.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+import {
+  DeepMap,
+  FieldError,
+  FieldValues,
+  UseFormRegister,
+} from 'react-hook-form'
+
+import { Status } from 'components/enums'
+import { LabeledInput } from 'components/molecules/labeled-input'
+
+import { UpdateFormEvent } from './types'
+
+const composeEvents =
+  (
+    eventA: (e: UpdateFormEvent) => void,
+    eventB?: (e: UpdateFormEvent) => void
+  ) =>
+  (...args: [UpdateFormEvent]): void => {
+    eventB && eventB(...args)
+    return eventA(...args)
+  }
+
+export const cloneElement = (
+  child: React.ReactElement,
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  register: UseFormRegister<any>,
```

Comment:
I think we can change `any` to `FieldValues` here and remove the eslint disable 🤔